doc/performance-investigations/0017-upgrade-python-3.9/summary.md

```diff
@@ -0,0 +1,30 @@
+# Upgrade Python from 3.8 to 3.9
+
+Python's latest minor release version is 3.9. EQ Census is currently running on version 3.8, whereas the post-census branch has been upgraded to 3.9. This looks at possible performance improvements due to the upgrade.
+
+## Benchmark profile
+
+| Option                 | Value                        |
+| ---------------------- | ---------------------------- |
+| Requests file          | census_household_gb_eng.json |
+| Run time               | 60m                          |
+| User wait time minimum | 1                            |
+| User wait time maximum | 2                            |
+| Clients                | 100                          |
+| Hatch rate             | 100                          |
+| Number of workers      | 7                            |
+| Number of threads      | 7                            |
+
+## Results
+
+Results based on 99th percentile timings.
+
+| Metric              | Baseline | Investigation |
+| ------------------- | -------- | ------------- |
+| Questionnaire GETs  | 198ms    | 154ms         |
+| Questionnaire POSTs | 299ms    | 250ms         |
+| All requests        | 247ms    | 202ms         |
```

Comment:
Can you use the outputs from get_summary script. Would be good to include the total requests served.

Reply:
Yeah sure thing